weave-js/src/components/Tooltip.tsx

```diff
@@ -24,6 +24,9 @@ type TooltipProps = {
   /** Element to be rendered in-place where the popup is defined. */
   trigger: React.ReactNode;
 
+  /** Value for Tooltip.Trigger `asChild` prop */
+  isTriggerAsChild?: boolean;
```

Comment:
for accessibility, tooltip trigger should have button role and tabindex so it can receive focus. by default, radix will wrap your trigger element for you to include these accessibility features. however, if your tooltip trigger is already a button, then we would want to use the `asChild` prop.

it looks like this `Tooltip` component was set up to always set `asChild` to true. however, there are cases where we just want to use radix's built-in accessibility features, so it would be more appropriate to set `asChild` to false.